src/search_algo.hpp

```diff
@@ -613,7 +619,8 @@ _setFrames(TBlastMatch                          & bm,
         bm.sFrameShift = (m.subjId % 3) + 1;
         if (m.subjId % 6 > 2)
             bm.sFrameShift = -bm.sFrameShift;
-    } else if constexpr (seqan::sHasRevComp(TLocalHolder::TGlobalHolder::blastProgram))
+    } else if constexpr (seqan::sHasRevComp(TLocalHolder::TGlobalHolder::blastProgram) ||
+                         TLocalHolder::TGlobalHolder::c_redAlph == AlphabetEnum::DNA3BS)
```

Comment:
Are you sure this change is correct? We have two frames on the subject, but there is never revcomp on the subject, is there?

Reply:
here we necode sFrameShift to mean second "translation" frame and not revcomp, or not?